embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceDeliveryCacheManager.kt

```diff
@@ -207,7 +208,7 @@ internal class EmbraceDeliveryCacheManager(
             }
             loadApiCallsResult.getOrNull() ?: loadPendingApiCallsOldVersion()
         }
-        val cached = backgroundWorker.submit(callable).get()
+        val cached = backgroundWorker.submit(TaskPriority.NORMAL, callable).get()
```

Comment:
Should we just rely on the default parameter when the task doesn't care about priority explicity?

Reply:
Yep this feels a reasonable thing to change, so that `TaskPriority` only needs to be specified when overriding the default.